### DISCUSSION TITLE
Created on 2 August 2023 at 00:17 [Sep 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6337] [Discord Threads]
> 🙋‍♂️ **.alex.pro**

**Comment by o1lab** [comment]
> 🙋‍♂️ **.alex.pro**  
#### https://records.cherishdev.com/conversations/2937c244-cda9-5634-a812-3475dadd6616 found the issue  
#  
> 🙋‍♂️ **.alex.pro**  
#### <@310157751395287040> https://github.com/nocodb/nocodb/issues/6111  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1136045413758156820)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
